frontend/src/store/modules/product/tables/index.js

```diff
@@ -141,13 +142,22 @@ const actions = {
     getTableSchema ({ commit }, { databaseId, tableName, teamId }) {
         return tablesApi.getTableSchema(teamId, databaseId, tableName)
             .then((schema) => {
+                schema.forEach(column => {
+                    column.safeName = hashString(column.name)
+                })
                 commit('setTableSchema', { databaseId, tableName, schema })
             })
     },
     getTableData ({ commit }, { databaseId, tableName, teamId }) {
         return tablesApi.getTableData(teamId, databaseId, tableName)
             .then((data) => {
-                commit('setTableData', { databaseId, tableName, data })
+                const payload = {
+                    data,
+                    safe: data.map(row => Object.fromEntries(
+                        Object.entries(row).map(([key, value]) => [hashString(key), value])
+                    ))
+                }
```

Comment:
Not 100% certain why the whole table data is hashed? Cant the key just be hashed when used?

Reply:
@cstns I wasnt clear. Why is every property name of every row being hashed? 
Perhaps a quick chat would clear this up?